WalletWasabi/Helpers/EndPointParser.cs

```diff
@@ -112,5 +111,10 @@ public static bool TryParse(string endPointString, int defaultPort, out EndPoint
 				return false;
 			}
 		}
+
+		private static bool IsValidPort(string port, out ushort p)
+		{
+			return ushort.TryParse(port, out p) && p <= IPEndPoint.MaxPort && p >= IPEndPoint.MinPort;
+		}
```

Comment:
As you mentioned in the description of this PR, the range comparisons make no sense here. After you remove them you will have tha `IsValidPort` is just a synonym of `ushort.TryParse` and then this is redundant.

Anyway, having synonyms sometimes make the code easier to read. I am not against that.